src/emcpy/calcs/wind.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 
+__all__ = ['uv_to_speed', 'spddir_to_uv']
```

Comment:
I suggest we change `uv_to_spddir` to be consistent with the adjoint operation.

Reply:
agreed, that makes sense.